### PR TITLE
Create ros_gz_spawn_model.launch

### PR DESCRIPTION
**Pull request by Amronos**
# 🎉 New feature

## Summary
Created `ros_gz_spawn_model.launch` taking inspiration from files like [ros_gz_sim.launch](https://github.com/gazebosim/ros_gz/blob/ros2/ros_gz_sim/launch/ros_gz_sim.launch).
Made to work after #606 is merged.

## Test it
Something like the following command can be used:
`ros2 launch ros_gz_sim ros_gz_spawn_model.launch world:=empty file:=$(ros2 pkg prefix --share ros_gz_sim_demos)/models/vehicle/model.sdf entity_name:=my_vehicle x:=5.0 y:=5.0 z:=0.5 bridge_name:=spawn_model_bridge config_file:=/path/to/config_file`

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: ros_gz_sim/launch/ros_gz_spawn_model.launch [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  <arg name="bridge_name" />
+  <arg name="config_file" />
+  <arg name="container_name" default="ros_gz_container" />
+  <arg name="namespace" default="" />
+  <arg name="use_composition" default="True" />
+  <arg name="use_respawn" default="False" />
+  <arg name="log_level" default="info" />
+  <arg name="world" default="" />
+  <arg name="file" default="" />
+  <arg name="string" default="" />
```
Comment (caguero) [inline ros_gz_sim/launch/ros_gz_spawn_model.launch]: Nit: I'd rename this parameter to `xml_string` to be consistent with `ros_gz_spawn_model.launch.py`.

Comment (Amronos) [reply ros_gz_sim/launch/ros_gz_spawn_model.launch]: I tried to be consistent with [these changes](https://github.com/gazebosim/ros_gz/pull/602#issuecomment-2336614296).

Comment (Amronos) [reply ros_gz_sim/launch/ros_gz_spawn_model.launch]: Changed it to `sdf_string`.

Comment (Amronos) [reply ros_gz_sim/launch/ros_gz_spawn_model.launch]: Changed it to `world_string`.

**Comment by caguero** [comment]
We'll have to wait to merge this patch right after the Gazebo Ionic release considering that isn't fixing a critical but.

**Comment by Amronos** [comment]
Should be ready to merge now.

**Comment by ahcorde** [comment]
https://github.com/Mergifyio backport jazzy

**Comment by Amronos** [comment]
#622 Should also be backported to Jazzy.

File: ros_gz_sim/launch/ros_gz_spawn_model.launch [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  <arg name="model_string" default="" />
+  <arg name="topic" default="" />
+  <arg name="entity_name" default="" />
+  <arg name="allow_renaming" default="False" />
+  <arg name="x" default="" />
+  <arg name="y" default="" />
+  <arg name="z" default="" />
+  <arg name="roll" default="" />
+  <arg name="pitch" default="" />
+  <arg name="yaw" default="" />
```
Comment (ahcorde) [inline ros_gz_sim/launch/ros_gz_spawn_model.launch]: default to `0.0` ?

**Review by ahcorde — CHANGES_REQUESTED** [review]

File: ros_gz_sim/launch/ros_gz_spawn_model.launch (hunk 2 above)
Comment (Amronos) [reply ros_gz_sim/launch/ros_gz_spawn_model.launch]: Thanks for pointing that out! Fixed in 6c0cbe7.

**Review by ahcorde — APPROVED** [review]

**Comment by ahcorde** [comment]
https://github.com/Mergifyio backport jazzy

**Pull request merged**